src/whatsnew/2.3.rst

```diff
@@ -0,0 +1,265 @@
+  services or domains alongside CouchDB.
+* The node-local (default port 5986) ``/_restart`` endpoint has been replaced by the
+  clustered (default port 5984) endpoint ``/_node/$node/_restart`` and
+  ``/_node/_local/_restart`` endpoints.
+* All python scripts shipped with CouchDB now specify and require Python 3.x.
+* :ghissue:`1396`: CouchDB is now compatible with Erlang 21.x.
+* :ghissue:`1680`: The embedded version of ``rebar`` used to build CouchDB has been
+  updated to the last version of ``rebar2`` available. This assists in building on
+  non-x86 platforms.
+
+.. _release/2.3.0:
+
+Version 2.3.0
+=============
+
+Features
+--------
+
+.. rst-class:: open
+
+* (Multiple) Clustered purge is now available. This feature restores the CouchDB 1.x
+  ability to completely remove any record of a document from a database. Conditions
+  apply; to use the feature safely, and for full details, read the complete
+  :ref:`cluster/purging` documentation.
+* :ghissue:`1658`: A new config setting is available, allowing an administrator to
+  configure an initial list of nodes that should be contacted when a node boots up.
+  Nodes in the ``seedlist`` that are successfully reached will be added to that node's
+  ``_nodes`` database automatically, triggering a distributed Erlang connection and
+  replication of the internal system databases to the new node. This can be used instead
+  of manual config or the cluster setup wizard to bootstrap a cluster. The progress of
+  the initial seeding of new nodes is exposed at the ``GET /_up`` endpoint.
+* Replication supports ipv6-only peers after updating ibrowse dependency.
+* :ghissue:`1543`: A CouchDB node can now be restarted via the ``/_node/$node/_restart``
+  or ``/_node/_local/_restart`` HTTP API endpoints. The node-local (port 5986)
+  ``/_restart`` endpoint has been removed.
+* :ghissue:`1708`: The UUID of the server/cluster is once again exposed in the
+  ``GET /`` response. This was a regression from CouchDB 1.x.
+* :ghissue:`1722`: Stats counts between job runs of the replicator are no longer reset
+  on job restart.
+* :ghissue:`1195`, :ghissue:`1742`: CouchDB's ``_bulk_get`` implementation now supports
+  the ``multipart/mixed`` and ``multipart/related`` content types if requested,
+  extending compatibility with third party replication clients.
+
+Performance
+-----------
+
+.. rst-class:: open
+
+* :ghissue:`1409`: CouchDB no longer forces the TCP receive buffer to a fixed size
+  of 256KB, allowing the operating system to dynamically adjust the buffer size. This
+  can lead to siginificantly improved network performance when transferring large
+  attachments.
+* :ghissue:`1423`: Mango selector matching now occurs at the shard level, reducing the
+  network traffic within a cluster for a mango query.
+* :ghissue:`1423`: Long running operations at the node level could exceed the inter-node
+  timeout, leading to a fabric timeout error in the logfile and a cancellation of the
```

Comment:
It's an internal library, there's no easier way to explain this. I left "fabric timeout" in there because that exact phrase shows up in the couch logfile if it happens.